docs/UserGuide.md

```diff
@@ -159,13 +159,25 @@ Examples:
 
 #### 3.1.2 Listing all clients : `listClient`
 
-List all clients in MyInsuRec.
+* List clients in MyInsuRec with a valid filter.
+* A valid filter can be clients who have bought the product `PRODUCT` or clients whose birthday is in range `BIRTHDAY`
+* `BIRTHDAY` is specified by keywords. The possible keywords are:
+    * `tomorrow` for a list of clients whose birthday is tomorrow;
+    * `week` for a list of clients whose birthday is in the next week;
+    * `month` for a list of clients whose birthday is in the next month.
 
-Format: `listClient`
+Format: `listClient [pd/PRODUCT] [bd/BIRTHDAY]`
```

Comment:
I think here should differentiate that either the user use product or birthday. In my PR,  I edited the `Features` and put `||` to differentiate.
example
`listClient [pd/PRODUCT] || [b/BIRTHDAY]`